src/js/vic-v2/config/form.js

```diff
@@ -74,6 +74,9 @@ const formConfig = {
               'ui:title': 'Gender',
               'ui:options': {
                 labels: genderLabels
+              },
+              'ui:errorMessages': {
+                required: 'Please select the response that most closely aligns with how you identify'
```

Comment:
Should this end with a period?

Reply:
This seems like it is also serving the purpose of a description, in addition to that of an error message.